### ISSUE TITLE
[Bug]:  Nginx not loading with Herd

### ISSUE THREAD

**Issue by Quaap70** [beyondcode/herd-community#60]
### Operating system version

MacOS Monterey 12.6.8

### System architecture

Intel (x86)

### Herd Version

1.0.8

### PHP Version

PHP 8.2

### Bug description

I removed php, brew and valet from this computer. Brew regularly gave me problems when updating packages. When updating one thing, something else breaks. And Herd seemed to me a more pleasant solution than Sail, because I could develop locally again. But Nginx won't start, the green indicator stays gray. I thought that all dependencies were installed with Herd, but apparently that is not the case? I tried installing pcre2 manually, but seems that I need Brew for that. I had just happily uninstalled brew.  ;-)

Can I fix this without needing Brew?  (Reinstalling macOS did not help)

(I also installed Herd on my desktop, which runs like a charm, and I'm very happy with it. Works really great and simple.)

### Relevant log output

```shell
dyld[1153] Library not loaded: ‘/usr/local/opt/pcre2/lib/libprce2-8.0.dylib’ Referenced from: ‘/Applications/Herd.app/Contents/Resources/nginx-x86'

Reason: tried: '/usr/local/opt/pcre2/lib/libprce2-8.0.dylib’ (no such file), '/usr/lib/libprce2-8.0.dylib' (no such file)
```

**Comment by mpociot** [comment]
Duplicate of #57 

TLDR: I'll recompile nginx for Intel and it'll be fixed in the next update (most likely out tomorrow)

**Comment by Quaap70** [comment]
Thanks Marcel!! I really love this App
